File: src/integration-tests/expecteds/py/boolean-schemas.py
from typing import TypedDict
from typing import Optional
from typing import NewType
from typing import Any
AlwaysTrue = NewType("AlwaysTrue", Any)
AlwaysFalse = NewType("AlwaysFalse", Any)

class ObjectOfCmepExQR(TypedDict):
    boolSchemaT: Optional[AlwaysTrue]
    boolSchemaF: Optional[AlwaysFalse]

File: src/integration-tests/expecteds/go/boolean-schemas.go
type AlwaysTrue interface{}
type AlwaysFalse interface{}
type ObjectOfCmepExQR struct {
	BoolSchemaT *AlwaysTrue  `json:"boolSchemaT,omitempty"`
	BoolSchemaF *AlwaysFalse `json:"boolSchemaF,omitempty"`
}
